.github/workflows/rsp-int-proj-tf.yaml

```diff
@@ -27,14 +27,18 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v3
 
-    # Gcloud CLI setup
+    # gcp auth
     - name: GCP login
-      uses: google-github-actions/setup-gcloud@v0
+      uses: google-github-actions/auth@v2
+      with:
+        credentials_json: ${{ secrets.PIPELINE_RSP_INT_PROJECT }}
```

Comment:
What are the rules by which we use `GOOGLE_CREDENTIALS` versus narrower secrets?  Or is it just, the newer environments got tighter-scoped credentials and as part of the overall eventual tech-debt modernization, we'd replace all the `GOOGLE_CREDENTIALS` with more limited creds?

Reply:
It would be part of tech debt modernization.  It could be part of the migration to Workload Identity.  I will note that in that ticket.  The process was started years ago and did not finish.